perftest/src/test/scala/simulations/BasicSimulation.scala

```diff
@@ -5,10 +5,16 @@ import io.gatling.core.Predef._
 import io.gatling.http.Predef._
 import scala.concurrent.duration._
 
+
 class BasicSimulation extends Simulation {
 
+  val registryUrl = scala.util.Properties.envOrElse("REGISTRY_URL", "http://localhost:8080/apis/registry/v1")
```

Comment:
do you want to add this changes in this PR? looks like part of your work on perf testing